src/Defra.PTS.Web.Application/Validation/PetKeeperAddressManualValidator.cs

```diff
@@ -2,49 +2,64 @@
-            RuleFor(x => x.TownOrCity).Matches(AppConstants.RegularExpressions.AddressText).WithMessage("Town or City is not valid");
-        });
+            RuleFor(x => x.AddressLineOne).NotEmpty().WithMessage("Enter line 1 of your address");
 
-        RuleFor(x => x.Postcode).NotEmpty().WithMessage(x => $"Enter postcode");
-        When(x => !string.IsNullOrWhiteSpace(x.Postcode), () =>
-        {
-            RuleFor(x => x.Postcode).Matches(AppConstants.RegularExpressions.UKPostcode).WithMessage("Postcode is not valid");
-            RuleFor(x => x.Postcode).MaximumLength(AppConstants.MaxLength.Postcode).WithMessage($"Postcode must be {AppConstants.MaxLength.Postcode} characters or less");
-            When(x => x.PostcodeStartsWithNonGBPrefix(), () =>
+            When(x => !string.IsNullOrWhiteSpace(x.AddressLineOne), () =>
             {
-                var validPostcode = false;
-                RuleFor(x => x.Postcode).Must(x => validPostcode).WithMessage("Enter a postcode in England, Scotland or Wales");
+                RuleFor(x => x.AddressLineOne)
+                    .MaximumLength(AppConstants.MaxLength.AddressLine).WithMessage($"Enter line 1 of your address using {AppConstants.MaxLength.AddressLine} characters or less")
+                    .Matches(AppConstants.RegularExpressions.AddressText).WithMessage($"Enter line 1 of your adderss using {AppConstants.MaxLength.AddressLine} characters or less ");
             });
-        });
 
-        When(x => !string.IsNullOrWhiteSpace(x.AddressLineTwo), () =>
-        {
-            RuleFor(x => x.AddressLineTwo).MaximumLength(AppConstants.MaxLength.AddressLine).WithMessage($"Address Line Two must be {AppConstants.MaxLength.AddressLine} characters or less");
-            RuleFor(x => x.AddressLineTwo).Matches(AppConstants.RegularExpressions.AddressText).WithMessage("Address Line Two is not valid");
-        });
+            RuleFor(x => x.TownOrCity)
+                .NotEmpty().WithMessage("Enter a town or city");
 
-        When(x => !string.IsNullOrWhiteSpace(x.County), () =>
-        {
-            RuleFor(x => x.County).MaximumLength(AppConstants.MaxLength.County).WithMessage($"County must be {AppConstants.MaxLength.County} characters or less");
-            RuleFor(x => x.County).Matches(AppConstants.RegularExpressions.AddressText).WithMessage("County is not valid");
-        });
+            When(x => !string.IsNullOrWhiteSpace(x.TownOrCity), () =>
+            {
+                RuleFor(x => x.TownOrCity)
+                    .MaximumLength(AppConstants.MaxLength.TownOrCity).WithMessage($"Enter a town or city using {AppConstants.MaxLength.TownOrCity} characters or less")
+                    .Matches(AppConstants.RegularExpressions.AddressText).WithMessage($"Enter a town or city using { AppConstants.MaxLength.TownOrCity} characters or less");
```

Comment:
Think it's just this one that will then need removing

Reply:
Removing the validation for the other fields due to the complexity of Welsh accented characters is understandable. It's good to know that we're still covered by the WAF for security. And since Welsh postcodes don't have accented characters, keeping the regex validation for that field is required.